charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -271,7 +271,7 @@ serviceMonitor:
   ##
   selfMonitor:
     enabled: false
-    additionalMetricsRelabels: {}
+    additionalMetricsRelabels: []
```

Comment:
thsi will break: https://github.com/prometheus-community/helm-charts/blob/e8b1efd4c1709a2c878d4e9923202ec303a897dc/charts/prometheus-blackbox-exporter/templates/selfpodmonitoring.yaml#L21

Reply:
@monotek can you please elaborate?

Reply:
you switch from list to map but map is used elsewhere.

Reply:
> but map is used elsewhere.

*list right ?

Reply:
> you switch from list to map but map is used elsewhere.

what do you suggest should I revert it to map again ?

Reply:
please checkout the latest commit I've reverted `additionalMetricsRelabels` to map.